src/nni_manager/training_service/pai/paiTrainingService.ts

```diff
@@ -497,12 +451,8 @@ class PAITrainingService implements TrainingService {
             trialJobDetail.paiJobName,
             // Docker image
             this.paiTrialConfig.image,
-            // dataDir
-            this.paiTrialConfig.dataDir,
-            // outputDir
-            this.paiTrialConfig.outputDir,
```

Comment:
so in the previous version, `outputDir` in nni config is also `outputDir` in pai job config?

Reply:
yes